python/paddle/fluid/contrib/slim/quantization/infer_quant_strategy.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import six
+import numpy as np
+import platform
+import os
+from .... import Executor
+from .... import io
+from .... import core
+from ....compiler import CompiledProgram
+from ....framework import IrGraph
+from ..core.strategy import Strategy
+import logging
+import sys
+
+__all__ = ['InferQuantStrategy']
+
+FORMAT = '%(asctime)s-%(levelname)s: %(message)s'
+logging.basicConfig(level=logging.INFO, format=FORMAT, stream=sys.stdout)
+logger = logging.getLogger(__name__)
+
+
+class InferQuantStrategy(Strategy):

```

Comment:
The name `InferQuantStrategy` and `infer_quant_strategy.py` are not better, since it means Post Training quantization strategy.

Reply:
how about 
`MKLDNNPostTrainingQuantStrategy`
`mkldnn_post_training_strategy.py`

Reply:
I think it is OK. @bingyanghuang how do you think about?

Reply:
> I think it is OK. @bingyanghuang how do you think about?

Looks fine to me.

Reply:
done